packages/ra-ui-materialui/src/detail/TabbedShowLayout.js

```diff
@@ -87,17 +87,19 @@ const TabbedShowLayout = props => {
         ...rest
     } = props;
     const match = useRouteMatch();
-
     const classes = useStyles(props);
+    const finalChildren = Array.isArray(children)
+        ? children.filter(child => child !== null)
+        : children;
 
     return (
         <div className={className} key={version} {...sanitizeRestProps(rest)}>
-            {cloneElement(tabs, {}, children)}
+            {cloneElement(tabs, {}, finalChildren)}
```

Comment:
what's the problem with cloning null children?

Reply:
Your suggestion passes the test but throw console errors since you end up with a different amount of headers and content if you have a `null` child